app/src/utils/birthday-utils.ts

```diff
@@ -0,0 +1,18 @@
+import { ClaimData } from "../types/claim";
+import moment from "moment";
+import { reformatDate } from "./formatter";
+
+export const check18Plus = (claim: ClaimData): boolean => {
+  if (claim.type === "DateOfBirthCredential") {
+    const userBirthday = reformatDate(claim.value.dob);
+    const age = moment(userBirthday, "YYYYMMDD").fromNow();
+    const userAge = age.split("").slice(0, 2).join("");
```

Comment:
Use a .substring

Reply:
We also have a year of birth utils.  We could have used that to get the year of birth.